modmesh/app/sample_mesh.py

```diff
@@ -29,14 +29,58 @@
 Show sample mesh
 """
 
-
 from .. import core
 from .. import view
+from .. import apputil
 
 
-def load_app():
-    cmd = "mh = mm.app.sample_mesh.run()"
-    view.app.pycon.command = cmd
+def help_tri(set_command=False):
```

Comment:
Decompose the `run()` function into several help prints so that we can interactively create multiple sub windows.